slothy/targets/arm_v81m/arch_v81m.py

```diff
@@ -109,14 +112,25 @@ def default_reserved():
         """Return the list of registers that should be reserved by default"""
         return set(["r14"])
 
-class Loop:
-
+class LeLoop(Loop):
+    """
+    Loop ending in a le instruction.
+    
+    Example:
+    ```
+           loop_lbl:
+               {code}
+               le <cnt>, loop_lbl
```

Comment:
The implementation is assuming that `cnt` increases by `1` per iteration, right?

Reply:
This can be broken through SW pipelining, or not, if the loop count increment is chosen as an early instruction?

Reply:
No, my fault -- this is already part of `LE`, I had forgotten.